lib/puppet/type/cisco_stp_global.rb

```diff
@@ -0,0 +1,755 @@
+# Manages the Cisco Spanning-tree Global configuration resource.
+#
+# Jan 2016
+#
+# Copyright (c) 2013-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+begin
+  require 'puppet_x/cisco/cmnutils'
```

Comment:
Not in this file.. you are using it in the provider, not the type.  I did add a note to consider moving it to the type though.